### PR TITLE
Soap as a disinfectant

### PR DESCRIPTION
**Pull request by Broken-Admin**
This makes soap a viable disinfectant for early-game players.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Being able to get rid of bites by rubbing them with a bar of regular soap heavily devalues both bites and all disinfection methods.
Plus, it kinda breaks the suspension of disbelief to not find soap everywhere. This is tolerable when the soap is nearly useless, but as soon as you are literally dying out of lack of soap, you start noticing its lack.

If the early bites are a problem, consider:
* Bumping disinfectant drops in easy to access locations (bathrooms)
* Bumping hydrogen peroxide drops in easy to access locations
* Making antibiotics spawn in much smaller stacks, but more commonly
* Adding some sort of grace period during early game when infections are less threatening due to character's "fresh" immune system, lack of adapted mutant bacteria, zombies having less adapted teeth etc.
* Reworking the bite infection formula to punish low armor much less
* Reworking the infection effects to do something less harmful to early game characters, but more to late game ones. Not sure what - mutations?
* gitting gud

**Comment by Broken-Admin** [comment]
I threw this together because an idea ChestHole had.

**Comment by kevingranade** [comment]
Feel free to ignore this:
> Being able to get rid of bites by rubbing them with a bar of regular soap heavily devalues both bites and all disinfection methods.

Though it does need to be "soap and water", not just soap.

**Comment by Coolthulhu** [comment]
Perfectly disinfecting deep bite wounds with soap and water, counteracted by soap being rarer than guns - yay for realism!

**Comment by Broken-Admin** [comment]
Could I add an liquid, *soapy water*? And let that be used for disinfection instead of soap alone?

File: data/json/items/armor.json [hunk 1]
```diff
@@ -1063,13 +1063,6 @@
     "flags": [ "BLIND" ]
   },
   {
-<<<<<<< HEAD
-    "copy-from": "blindfold",
-    "material": [ "plastic", "cotton" ]
-  },
-  {
-=======
->>>>>>> parent of 6cc31694b9... Added blindfold_tape
```
Comment (Soyweiser) [inline data/json/items/armor.json]: "What was that?"
"->>>>>>> parent of 6cc31694b9... Added blindfold_tape"
Those are messages about the difference between the two files. No idea how you managed to include those.

**Comment by Broken-Admin** [comment]
Trying to revert the files.

File: data/json/items/armor.json (hunk 1 above)
Comment (Broken-Admin) [reply data/json/items/armor.json]: By trying to revet to a previous commit.

**Comment by Broken-Admin** [comment]
@kevingranade Soapy water, or new item action?

File: data/json/items/armor.json (hunk 1 above)
Comment (Soyweiser) [reply data/json/items/armor.json]: There should be a command for that, you shouldn't do that by hand, unless a command added that, which is even more confusing to me.

**Comment by kevingranade** [comment]
New item action makes the most sense, activate soap with a source of water nearby to scrub your wound.

**Comment by Night-Pryanik** [comment]
Honestly I agree with @Coolthulhu here. Soapy water can help only with skin-deep scratches, not with deep bites. Soap's not a disinfectant, it is used to just rinse pathogens off the wound.

**Comment by kevingranade** [comment]
Cleaning the wound is the primary treatment for bite wounds irl.  Strong antiseptics are not recommended for open wounds due to damage it can cause to surrounding tissue.

Please, read up on first aid a bit before opining about what is and is not reasonable wound treatment.

**Comment by tinukedaya** [comment]
Should not that be a **clean water** source? 
I mean using water from a swamp or that old standing water from toilets, even with soap is probably not the best thing to do on open wound... It would probably do more harm than good. It's not filthy cloth you are trying to clean up here.

**Comment by Night-Pryanik** [comment]
Cleaning wound with clean water is only a primary treatment, not a cure. It can't neutralize infections at all.

**Comment by AMurkin** [comment]
Can't agree that soap may be used to cure infections, too. With soap, but zero immunity bited gonna be dead soon.

**Comment by tinukedaya** [comment]
No what I mean is for using soap as treatment you **additionally** need a clean water source. Not just water.

**Comment by kevingranade** [comment]
> Cleaning wound with clean water is only a primary treatment, not a cure.

So is using an antiseptic.  Cleaning with alcohol doesnt "cure" infections, it just cleans infectious matter from the wound to try and prevent an infection.

**Comment by Night-Pryanik** [comment]
> So is using an antiseptic. Cleaning with alcohol doesnt "cure" infections, it just cleans infectious matter from the wound to try and prevent an infection.

Nope, antiseptic indeed does "cures infection". Quote from the Wiki: 
> Some antibiotics are true germicides, capable of destroying microbes (bacteriocidal), while others are bacteriostatic and only prevent or inhibit their growth.

**Comment by Treah** [comment]
I agree that soap + clean water should be allowed to clean bites but not cure infections ( BLUE on the status screen) This would be realistic and a good use for soap. However I would add that this should also be added to the additional things used for cleaning such as detergent.

**Comment by Soyweiser** [comment]
@Night-Pryanik you mean wikipedia wiki right? And not the CDDA wiki?

**Comment by Night-Pryanik** [comment]
@Soyweiser yeah, wikipedia.

File: data/json/items/armor.json (hunk 1 above)
Comment (Broken-Admin) [reply data/json/items/armor.json]: It was done by the command...

**Comment by Broken-Admin** [comment]
Oh lord, wish me luck on item use. This'll be something new.

**Comment by Broken-Admin** [comment]
Going to close and reopen later when I'm ready for a new project.

**Pull request closed without merging**